GitUI/CommitInfo/CommitInfo.cs

```diff
@@ -594,5 +677,14 @@ private void _RevisionHeader_ContentsResized(object sender, ContentsResizedEvent
             _headerResize = e.NewRectangle;
         }
 
+        private void commitSignPicture_Click(object sender, EventArgs e)
+        {
+            DialogResult res = MessageBox.Show(Module.GetGpgRawVerificationMessage(_revision.Guid), "GPG Commit Info", MessageBoxButtons.OK);
+        }
+
+        private void tagSignPicture_Click(object sender, EventArgs e)
+        {
+            DialogResult res = MessageBox.Show(tagGpgMessage, "GPG Tag Info", MessageBoxButtons.OK);
```

Comment:
All translatable strings need to be defined as `TranslationString`. 
Please have a look in other forms how they are done.